docs/examples/extend/create_a_custom_operator.ipynb

```diff
@@ -186,6 +186,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from __future__ import print_function\n",
```

Comment:
I don't know if we need that one. I think Jupyter deals with that without that import, doesn't it?

Reply:
They seem to be using it in their own doc, maybe it's safe to have it anyway https://jupyter-notebook.readthedocs.io/en/stable/examples/Notebook/Running%20Code.html#sys.stdout-and-sys.stderr

Reply:
removed